tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -9,7 +9,13 @@ load "${BATS_TEST_DIRNAME}/lib.sh"
 load "${BATS_TEST_DIRNAME}/confidential_common.sh"
 
 setup() {
-    confidential_setup && skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
+    if is_confidential_hardware; then
+        skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
+    fi
+
+    if is_confidential_runtime_class; then
```

Comment:
I think this check is wrong?
```suggestion
    if ! is_confidential_runtime_class; then
```

Same for @test and teardown versions in this file?

Reply:
`@test` is an user in github and its pic is a guitar players :D

Reply:
Sorry to disturb you Mr Test!

Reply:
Updated!